#! /bin/sh /usr/share/dpatch/dpatch-run
## A strange copy operation of LLStyle.dpatch by  <root@debian.example.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Patch by Alissa Sabre

@DPATCH@

Index: linden/indra/llui/llstyle.cpp
===================================================================
--- linden/indra/llui/llstyle.cpp	(revision 53)
+++ linden/indra/llui/llstyle.cpp	(working copy)
@@ -112,7 +112,7 @@
 		mItalic = rhs.mItalic;
 		mBold = rhs.mBold;
 		mUnderline = rhs.mUnderline;
-		mDropShadow = rhs.mUnderline;
+		mDropShadow = rhs.mDropShadow;
 		mIsEmbeddedItem = rhs.mIsEmbeddedItem;
 	}
 	
